.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
 python:
   - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
```

Comment:
These are redundant - we're already running the tests on Python 2.6-3.4 using tox – see the matrix in https://github.com/toastdriven/django-haystack/blob/74b9eba6fb572f6b3873a833535dfa42fc992795/.travis.yml#L29-L42